intelligent-service/src/main.py

```diff
@@ -1,32 +1,40 @@
 """
 Main file for server
 """
-import numpy as np
+import os
 import flask
 from quadraticcontroller import quadratic_api
 from generalcontroller import general_api
 
+
 # Press Shift+F10 to execute it or replace it with your code.
 # Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
 
-app = flask.Flask(__name__)
-app.config["DEBUG"] = True
-app.config['JSON_SORT_KEYS'] = False
-app.register_blueprint(quadratic_api)
-app.register_blueprint(general_api)
-
-def print_hi(name):
-    """ Docstring """
-    # Use a breakpoint in the code line below to debug your script.
-    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
 
+def read_config():
```

Comment:
assert is for debuging and testing usually. 

https://tech.serhatteker.com/post/2020-02/test-env-vars-in-python/

The get_secret function seems to be right. 

Use one function per environment variable. 

read config will be like the following:

def read_config():
return read_port(), read_info_sys_host(), read_etc();